astropy/utils/data.py

```diff
@@ -1,16 +1,17 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-""" This module contains helper functions for accessing, downloading, and
-caching data files.
-"""
+"""Functions for accessing, downloading, and caching data files."""
 
 import atexit
 import contextlib
+import dbm
 import fnmatch
 import hashlib
 import os
 import io
+import json
```

Comment:
My Emails says `import json` is unused. If that is true, please remove it. Thanks!